shared/src/commonMain/kotlin/org/hyperskill/app/home/injection/HomeComponentImpl.kt

```diff
@@ -33,5 +35,5 @@ class HomeComponentImpl(private val appGraph: AppGraph) : HomeComponent {
             gamificationToolbarComponent.gamificationToolbarActionDispatcher,
             topicsToDiscoverNextComponent.topicsToDiscoverNextReducer,
             topicsToDiscoverNextComponent.topicsToDiscoverNextActionDispatcher
-        )
+        ).transformState(HomeFeatureViewStateMapper::map)
```

Comment:
I think this is an artifact from when merging the remote origin branch, please revert this change.

This line of code already exists in the `HomeFeatureBuilder.build`